app/controllers/health_monitor/health_controller.rb

```diff
@@ -10,10 +10,9 @@ class HealthController < ActionController::Base
     def check
       res = HealthMonitor.check(request: request)
 
-      unless HealthMonitor.configuration.environmet_variables.nil?
```

Comment:
I mean, calling `merge` from `nil` will result in `NoMethodError: undefined method merge for nil:NilClass`, no?